### ISSUE TITLE
Error while minifying app code in fs-extra

### ISSUE THREAD

**Issue by achtan** [meteor/meteor#8542]
hi, im getting this error while running `meteor run --production`
````
=> Errors prevented startup:

   While minifying app code:
   packages/minifyStdJS_plugin.js:113:28: UglifyJS minification error:

   SyntaxError: Unexpected token: operator (>) at node_modules/fs-extra/lib/index.js line 9

   within packages/modules.js (line: 167003, col: 30, pos: 19253373):

   Object.keys(gfs).forEach(key => {

   at maybeThrowMinifyErrorBySourceFile (packages/minifyStdJS_plugin.js:113:28)
   at packages/minifyStdJS_plugin.js:145:9
   at Array.forEach (native)
   at UglifyJSMinifier.processFilesForBundle (packages/minifyStdJS_plugin.js:130:9)
````

node_modules/fs-extra/lib/index.js line 8-11
````js
// attach fs methods to fse
Object.keys(gfs).forEach(key => {
  fse[key] = gfs[key]
})
````

METEOR@1.4.3.2

````
accounts-base@1.2.15
accounts-password@1.3.4
alanning:roles@1.2.16
allow-deny@1.0.5
autoupdate@1.3.12
babel-compiler@6.14.1
babel-runtime@1.0.1
base64@1.0.10
bensventures:flow-router-ssr@3.13.1
binary-heap@1.0.10
blaze@2.3.2
blaze-html-templates@1.1.2
blaze-tools@1.0.10
boilerplate-generator@1.0.11
caching-compiler@1.1.9
caching-html-compiler@1.1.2
callback-hook@1.0.10
check@1.2.5
chuangbo:cookie@1.1.0
coffeescript@1.11.1_4
ddp@1.2.5
ddp-client@1.3.3
ddp-common@1.2.8
ddp-rate-limiter@1.0.7
ddp-server@1.3.13
deps@1.0.12
diff-sequence@1.0.7
ecmascript@0.6.3
ecmascript-runtime@0.3.15
ejson@1.0.13
email@1.1.18
es5-shim@4.6.15
fastclick@1.0.13
fortawesome:fontawesome@4.7.0
fourseven:scss@3.13.0
geojson-utils@1.0.10
hot-code-push@1.0.4
html-tools@1.0.11
htmljs@1.0.11
http@1.2.12
id-map@1.0.9
jquery@1.11.10
kadira:dochead@1.5.0
launch-screen@1.1.1
less@2.7.9
livedata@1.0.18
localstorage@1.0.12
logging@1.1.17
matb33:collection-hooks@0.8.4
meteor@1.6.1
meteor-base@1.0.4
meteorhacks:aggregate@1.3.0
meteorhacks:collection-utils@1.2.0
meteorhacks:meteorx@1.4.1
meteorhacks:picker@1.0.3
minifier-css@1.2.16
minifier-js@1.2.18
minimongo@1.0.21
mobile-experience@1.0.4
mobile-status-bar@1.0.14
modules@0.7.9
modules-runtime@0.7.9
mongo@1.1.16
mongo-id@1.0.6
mongo-livedata@1.0.12
npm-bcrypt@0.9.2
npm-mongo@2.2.24
observe-sequence@1.0.16
ordered-dict@1.0.9
ostrio:cookies@2.2.0
ostrio:files@1.7.13
percolate:migrations@0.9.8
promise@0.8.8
random@1.0.10
rate-limit@1.0.7
reactive-dict@1.1.8
reactive-var@1.0.11
reload@1.1.11
retry@1.0.9
routepolicy@1.0.12
service-configuration@1.0.11
session@1.1.7
sha@1.0.9
softwarerero:accounts-t9n@1.3.9
spacebars@1.0.15
spacebars-compiler@1.1.2
srp@1.0.10
standard-minifier-css@1.3.4
standard-minifier-js@1.2.3
staringatlights:fast-render@2.16.3
staringatlights:inject-data@2.0.5
std:accounts-bootstrap@1.2.0
std:accounts-ui@1.2.19
templating@1.3.2
templating-compiler@1.3.2
templating-runtime@1.3.2
templating-tools@1.1.2
tmeasday:check-npm-versions@0.3.1
tracker@1.1.2
ui@1.0.13
underscore@1.0.10
url@1.1.0
webapp@1.3.14
webapp-hashing@1.0.9
````
npm
````
  "dependencies": {
    "antd": "^2.7.2",
    "babel-runtime": "^6.18.0",
    "bcrypt": "^1.0.2",
    "bootstrap": "^4.0.0-alpha.6",
    "classnames": "^2.2.5",
    "domready": "^1.0.8",
    "enumify": "^1.0.4",
    "faker": "^3.1.0",
    "fs-extra": "^2.1.2",
    "jimp": "^0.2.27",
    "lodash": "^4.17.4",
    "meteor-node-stubs": "^0.2.4",
    "mobx": "^3.0.1",
    "mobx-react": "^4.1.0",
    "numeral": "^2.0.4",
    "react": "^15.4.2",
    "react-addons-css-transition-group": "^15.4.2",
    "react-addons-shallow-compare": "^15.4.2",
    "react-addons-transition-group": "^15.4.2",
    "react-dnd": "^2.1.4",
    "react-dnd-touch-backend": "^0.3.5",
    "react-dom": "^15.4.2",
    "react-dropzone": "^3.12.2",
    "react-google-maps": "^6.0.1",
    "react-image-gallery": "^0.7.12",
    "react-komposer": "^2.0.0",
    "react-redux": "^5.0.2",
    "react-selectize": "^2.1.0",
    "react-waypoint": "^5.0.3",
    "reactstrap": "^4.0.1",
    "redux": "^3.6.0",
    "redux-form": "^6.4.3",
    "tracker-component": "^1.3.21"
  },
  "devDependencies": {
    "@kadira/storybook": "^2.21.0",
    "babel-plugin-import": "^1.1.1",
    "babel-plugin-transform-class-properties": "^6.23.0",
    "babel-plugin-transform-decorators-legacy": "^1.3.4",
    "babel-preset-es2015": "^6.9.0",
    "babel-preset-es2016": "^6.11.3",
    "babel-preset-react": "^6.11.1",
    "babel-preset-stage-1": "^6.13.0",
    "babel-preset-stage-2": "^6.18.0",
    "babel-root-slash-import": "^1.1.0",
    "node-sass": "^4.5.0",
    "sass-loader": "^6.0.2"
  }
````

**Comment by abernix** [comment]
Could you try this with Meteor 1.4.4-rc.2 (a recent release candidate, but not an official release)?  This should be fixed by 0d657ed24a8b0073f904fbcded7368b186db0e32.

    meteor update --release 1.4.4-rc.2

**Comment by achtan** [comment]
on 1.4.4 i have different issue :) #8537
i fixed this one with using `abernix:standard-minifier-js@1.3.19` instead of `standard-minifier-js`

**Comment by abernix** [comment]
@achtan If you have a moment, I'd still be curious if this _particular_ problem goes away if you update to Meteor `1.4.4-rc.2` (or `1.4.4-rc.4`, released last night).  I'm glad `abernix:standard-minifier-js` fixed it for you, but we're hoping that 1.4.4 will do that too, so the confirmation would be nice!

Also, pretty please, [provide a reproduction on #8537](https://github.com/meteor/meteor/issues/8537#issuecomment-290260697)? :)
